src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px 0;
+}
+
+.list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+  line-height: 60px;
```

Comment:
```suggestion
```
Use vertical paddings for links instead

Reply:
![image](https://user-images.githubusercontent.com/97547824/205701862-747cc7a6-0c72-4de4-8eb4-9937387de7b3.png)
Didn't fixed